jina/peapods/compoundpod/helper.py

```diff
@@ -0,0 +1,186 @@
+import copy
```

Comment:
Is this just C&P from the regular pod? If so, please let's de-duplicate code.

Reply:
In the first iteration, I wanted to keep the existing pod logic as it is and refactor it in a follow-up ticket.
That would make the review process easier. If you want, I can also combine the code.